internal/http/services/owncloud/ocs/conversions/role.go

```diff
@@ -154,6 +165,15 @@ func NewUnknownRole() *Role {
 	}
 }
 
+// NewDeniedRole creates a fully denied role
+func NewDeniedRole() *Role {
+	return &Role{
+		Name:                   RoleDenied,
+		cS3ResourcePermissions: &provider.ResourcePermissions{},
```

Comment:
@butonic @kobergj we need to check how the decomposedFS works with these permissions. The idea is that this node and all childs should not have any permissions for the grantee. Maybe we need to implement it.

Reply:
Should work imo. Decomposedfs checks for specific `ResourcePermissions` before doing anything. In this case `ResourcePermissions` would be all false.

Reply:
Ok, so that works for the whole tree below that node? Let me test it.